daemon/data_manager.go

```diff
@@ -385,3 +387,22 @@ func (dm *DataManager) CountryCodeToCountryName(code string) string {
 
 	return ""
 }
+
+func (dm *DataManager) SetAccountData(accountData *pb.AccountResponse) {
+	dm.mu.Lock()
+	defer dm.mu.Unlock()
+
+	dm.accountData.set(accountData)
+}
+
+func (dm *DataManager) GetAccountData() (*pb.AccountResponse, bool) {
+	dm.mu.Lock()
+	defer dm.mu.Unlock()
+	return dm.accountData.get()
+}
+
+func (dm *DataManager) InvalidateAccountdata() {
```

Comment:
There is small inconsistency between `Accountdata` here and `AccountData` in other methods